index.html

```diff
@@ -10,229 +10,233 @@
+                </p>
+            </div>
+
+            <ul class="network">
+                <li class="social">
+                    <a target="_blank" href="https://github.com/Kakalanp" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/github.png" alt="g"/></a>
+                </li>
+                <li class="social">
+                    <a target="_blank" href="https://www.linkedin.com/in/joaquin-garrido-lecca-zanetti-623583204/" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2--v1.png" alt="li"/></a>
+                </li>
+                <li class="social">
+                    <a target="_blank" href="https://www.twitter.com/LeccaJoaquin"  rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/twitter--v1.png" alt="t"/></a>
+                </li>
+            </ul>
         </section>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Javascript</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
-        </section>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
+        <section class="works">
+            <div id="works-title">
+                <h2>My recent works</h2>
+                <img src="/Images/works-horizontal-line.svg" alt="line">
+            </div>
             
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
-        </section>
+            <div class="works">
+                <div class="card">
+                    <div class="screenshot">
+                        <img src="/Images/placeholder.png" alt="screenshot">
+                    </div>
+                    
+                    <div class="info">
+                        <h3 class="text">
+                            Multi-Post Stories Gain+Glory
+                        </h3>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
+                        <a class="button" href="">See project</a>
+                    </div>
+                </div>
+    
+                <div class="card">
```

Comment:
- [ ] As stated by the previous reviewer, please reduce the space between the cards so it matches the given design.  This space is a little too much, and does not match the required Figma design:

![image](https://user-images.githubusercontent.com/57421823/149531955-bd6bb9af-5123-4ca5-9e0c-08f2b687054f.png)

Kindly make this change so your project looks like the design, as stated in the requirements of the project 💯